tests/check_framework/urls/unique_namespaces.py

```diff
@@ -0,0 +1,16 @@
+from django.conf.urls import include, url
+
+common_url_patterns = [
+    url(r'^app-ns1/', include(([], 'app'))),
```

Comment:
One of these should have an explicit instance namespace other than `'app'`, otherwise the nested namespaces are not unique.